openquake/qa_tests_data/classical/case_65/expected/full_ruptures.csv

```diff
@@ -1,3 +1,5 @@
+1483155043,5.200000E+00,9.000000E+01,9.55067,45.27761,6.019526E+00,10,Active Shallow Crust,NonParametricProbabilisticRupture KiteSurface,"[[[[10.0, 9.93633, 9.87328, 9.80961, 9.74657, 9.6829, 9.61923, 9.55618], [9.96392, 9.9007, 9.83684, 9.77361, 9.71038, 9.64651, 9.58328, 9.52004], [9.92782, 9.86439, 9.80095, 9.7375, 9.67405, 9.61059, 9.54713, NaN], [9.8917, 9.82865, 9.76495, 9.70188, 9.63817, 9.57508, 9.51199, NaN], [9.85555, 9.79255, 9.72953, 9.6665, 9.60346, 9.54041, NaN, NaN], [9.81939, 9.75689, 9.69438, 9.63124, 9.5687, 9.50613, NaN, NaN], [9.78319, 9.72119, 9.65916, 9.59773, 9.53566, NaN, NaN, NaN], [9.74698, 9.6866, 9.6256, 9.56516, 9.50411, NaN, NaN, NaN], [9.71074, 9.65241, 9.59346, 9.53506, NaN, NaN, NaN, NaN], [9.67448, 9.61963, 9.56419, 9.50925, NaN, NaN, NaN, NaN], [9.63819, 9.58856, 9.53938, NaN, NaN, NaN, NaN, NaN], [9.60188, 9.56132, 9.5211, NaN, NaN, NaN, NaN, NaN], [NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN], [NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN]], [[45.0, 45.00012, 45.00021, 45.00026, 45.00027, 45.00026, 45.0002, 45.00011], [45.03645, 45.03761, 45.03874, 45.03982, 45.04087, 45.0419, 45.04287, 45.04382], [45.07291, 45.07528, 45.07761, 45.07992, 45.08218, 45.08442, 45.08661, NaN], [45.10936, 45.11315, 45.11694, 45.12066, 45.12438, 45.12803, 45.13165, NaN], [45.14581, 45.15132, 45.1568, 45.16224, 45.16765, 45.17302, NaN, NaN], [45.18226, 45.18983, 45.19736, 45.20493, 45.2124, 45.21983, NaN, NaN], [45.21872, 45.22884, 45.23894, 45.2489, 45.25892, NaN, NaN, NaN], [45.25517, 45.26833, 45.28159, 45.29469, 45.30789, NaN, NaN, NaN], [45.29162, 45.30863, 45.32578, 45.34273, NaN, NaN, NaN, NaN], [45.32807, 45.34979, 45.37169, 45.39336, NaN, NaN, NaN, NaN], [45.36452, 45.39207, 45.41932, NaN, NaN, NaN, NaN, NaN], [45.40098, 45.43456, 45.4678, NaN, NaN, NaN, NaN, NaN], [NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN], [NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN]], [[0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.72754, 0.76041, 0.79394, 0.82713, 0.86033, 0.89386, 0.92706, 0.96025], [1.45508, 1.52663, 1.59889, 1.67115, 1.74342, 1.81568, 1.88794, NaN], [2.18262, 2.29917, 2.41807, 2.53579, 2.65469, 2.77241, 2.89014, NaN], [2.91016, 3.08133, 3.25417, 3.42701, 3.59985, 3.77269, NaN, NaN], [3.6377, 3.87405, 4.11274, 4.35377, 4.59246, 4.83115, NaN, NaN], [4.36524, 4.68276, 5.00346, 5.32097, 5.64167, NaN, NaN, NaN], [5.09278, 5.5067, 5.92874, 6.34671, 6.76875, NaN, NaN, NaN], [5.82032, 6.3562, 6.90289, 7.44418, NaN, NaN, NaN, NaN], [6.54786, 7.23317, 7.93217, 8.62433, NaN, NaN, NaN, NaN], [7.2754, 8.14602, 9.01663, NaN, NaN, NaN, NaN, NaN], [8.00293, 9.06551, 10.1281, NaN, NaN, NaN, NaN, NaN], [NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN], [NaN, NaN, NaN, NaN, NaN, NaN, NaN, NaN]]]]","{""occurrence_rate"": NaN}"
```

Comment:
Maybe we should remove these NaNs from output ... unless this is a matrix with a given shape

Reply:
No, I need to export exactly what is in the datastore, otherwise the rupture geometry cannot be reconstructed.

Reply:
Fine. That's what I supposed.